back/src/chat/channel/entities/channel.entity.ts

```diff
@@ -5,7 +5,6 @@ import { Column, Entity, JoinTable, ManyToMany, ManyToOne, OneToMany, OneToOne,
 enum ChanType {
 	Public,
 	Private,
```

Comment:
I think we should put back protected --- I thought Protected was optional and we could do Public and Private OR Protected, but according to Simon, Private and Protected channels are two different things , Private can only be joined with an invite (password is not mandatory) and Protected can be joined without an invite but with a password (so anyone who knows the password can join without an invite).